### ISSUE TITLE
sendAction deprecation

### ISSUE THREAD

**Issue by josemarluedke** [Addepar/ember-table#614]
Running ember-table with Ember v3.4 beta gave me the following deprecation:

```
DEPRECATION: You called <my-app@component:ember-tr::ember7583>.sendAction("onClick") but Component#sendAction is deprecated. Please use closure actions instead. [deprecation id: ember-component.send-action] See https://emberjs.com/deprecations/v3.x#toc_ember-component-send-action for more details.
```

**Comment by pzuraq** [comment]
Unfortunately due to our support matrix (we support Ember 1.12+) we won’t be able to change this API until Ember Table v3. We should look to see if there is a way to suppress this deprecation until then.

**Comment by Alonski** [comment]
Isn't there a way to check what version the app is running and call functions accordingly? 
Do closure actions and legacy actions at the same time somehow?

**Comment by pzuraq** [comment]
There is [ember-compatability-helpers](https://github.com/pzuraq/ember-compatibility-helpers) which would allow you to do that, happy to merge PRs but I won’t have time to continue working on this project for the forseeable future. Main thing that will be tricky is tests

**Comment by Alonski** [comment]
@pzuraq That looks interesting. If I find the time I will check it out. Thanks

**Comment by HeroicEric** [comment]
I believe this was fixed by https://github.com/Addepar/ember-table/pull/860, so this can probably be closed

**Comment by kpfefferle** [comment]
Resolved by #860
